index.html

```diff
@@ -201,6 +204,15 @@ <h2 id="constraints">Constraining Display Surface Selection</h2>
         the <a>visible display surface</a>.</p>
 
         <dl class="idl" title="partial dictionary MediaTrackConstraintSet">
+          <dt>ConstrainBoolean disableLocalEcho</dt>
+
+          <dd>
+            <p>A value of <code>true</code> indicates that we mute the local
+            playback for the captured content. For example, when capturing a
+            browser tab, the tab will be muted then; when capturing the full
+            screen with audio, then the system audio will be muted then.</p>
```

Comment:
Grammar nit: I think you can delete all occurences of "then" in this sentence and make it more readable.